api/preprints/views.py

```diff
@@ -82,7 +87,7 @@ class PreprintList(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixin):
 
     + `filter[<fieldname>]=<Str>` -- fields and values to filter the search results on.
 
-    Preprints may be filtered by their `id`, `title`, `public`, `tags`, `date_created`, `date_modified`, `provider`, and `subjects`
+    Preprints may be filtered by their `id`, `title`, `public`, `tags`, `date_created`, `date_modified`, and `subjects`
```

Comment:
No, this is fine. The only thing filter would add would be `/v2/preprints/?filter[provider]=rxiv` to get non-osf providers, but I'm pretty sure that's not a required feature right now.